File: python/XRPL_UKL1/soal2.py
#!/bin/python3
"""
Soal    : 2
A       : 5
B       : 6
"""

A=5
B=9 
u=A
s=A 
print("Deret Aritmatikanya adalah")
for i in range(5):                        
    for j in range(6):
        print(u, end="\t"); 
        u=u+B;
        s=s+u;
    print(); 

s=s-u
print(f"Jumlah Deret Aritmatikanya adalah {s}"); 

File: js/XRPL_UKL1/soal2.js
/*
Soal    : 2
A       : 5
B       : 6
*/

A=5;
B=9;
u=A; 
s=A; 

console.log("Deret Aritmatikanya adalah"); 

for(i=0;i<5;i++){                            
    for(j=0;j<6;j++){  
        process.stdout.write(u+"\t"); 
        u=u+B;
        s=s+u;
    }
    console.log(); 
}
s=s-u;
console.log("Jumlah Deret Aritmatikanya adalah "+s); 